leap/example.rb

```diff
@@ -1,25 +1,12 @@
 class Year
```

Comment:
Any particular reason why you changed the program to use class methods instead of delegating to the instance?